Azure Pipeline Tests "failed" with HTTP request timeout

OSX run failed: https://github.com/zkSNACKs/WalletWasabi/runs/15165893562
Azure logs: https://dev.azure.com/zkSNACKs/Wasabi/_build/results?buildId=102354&view=logs&j=a5e52b91-c83f-5429-4a68-c246fc63a4f7&t=d4f393a6-5f7f-57b8-9785-cfc6593dde33

You can see in the logs that
```lua
Test Run Successful.
Total tests: 1087
     Passed: 1087
 Total time: 15.0790 Minutes
```
But when Azure tried to upload/publish the results
```lua
Async Command Start: Publish test results
Async Command End: Publish test results
##[error]The HTTP request timed out after 00:00:30.
Finishing: Test Debug
```
it got timed out, thus indicating that the tests have faild, even though they didn't.
![image](https://github.com/zkSNACKs/WalletWasabi/assets/54551879/28db9585-d20f-4d0d-a231-c564ec849b37)
You can see that everything else went smooth. Probably a one time thing on Azure's side, but can we do something that in this case, it should say that the tests were successfull anyway?

--- a/azure-pipelines-linux.yml
+++ b/azure-pipelines-linux.yml
@@ -30,11 +30,6 @@ jobs:
     inputs:
       command: build
       arguments: --no-restore --configuration Release
-  - task: CmdLine@2
-    inputs:
-      # Remove PathMap properties which ensure deterministic builds, since this
-      # interferes with the coverage analysis.
-      script: 'sed -i "/PathMap/d" $(System.DefaultWorkingDirectory)/*/*.csproj'
   - task: DotNetCoreCLI@2
     displayName: 'Test Debug'
     inputs:

--- a/azure-pipelines-osx.yml
+++ b/azure-pipelines-osx.yml
@@ -30,12 +30,6 @@ jobs:
     inputs:
       command: build
       arguments: --no-restore --configuration Release
-  - task: CmdLine@2
-    inputs:
-      # Remove PathMap properties which ensure deterministic builds, since this
-      # interferes with the coverage analysis. sed -i seems unhappy with the
-      # shell glob but perl doesn't mind.
-      script: 'perl -ni -e "print unless /PathMap/" $(System.DefaultWorkingDirectory)/*/*.csproj'
   - task: DotNetCoreCLI@2
     displayName: 'Test Debug'
     inputs:

